src/constants.tsx

```diff
@@ -3,13 +3,15 @@ import { Icon } from '@iconify/react'
 import noteIcon from '@iconify/icons-simple-line-icons/note'
 import NeoConvertor from 'neo-convertor'
 
+import './components/navigation/Sidebar.scss'
 import { store } from './store'
 import { ReactComponent as Home } from './assets/icons/home.svg'
 import { ReactComponent as Transactions } from './assets/icons/transactions.svg'
 import { ReactComponent as Blocks } from './assets/icons/blocks.svg'
 import { ReactComponent as Wallets } from './assets/icons/wallets.svg'
 import { ReactComponent as Api } from './assets/icons/api.svg'
 import { ReactComponent as Magnify } from './assets/icons/magnify.svg'
+import Monitor from './assets/icons/monitor.png'
```

Comment:
Please use an svg here (replicate the pattern used for every other route)

Reply:
https://cityofzion.slack.com/archives/C01LS9M9A84/p1618009962000100

Reply:
I just fixed it in #277 
Once it is approved there, we'll make the rebase here.